src/spread/mult-obj-null.case

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: Object Spread operator following other arguments with null value
+template: default
+esid: pending
+info: |
+    Pending Runtime Semantics: PropertyDefinitionEvaluation
+
+    PropertyDefinition:...AssignmentExpression
+
+    1. Let exprValue be the result of evaluating AssignmentExpression.
+    2. Let fromValue be GetValue(exprValue).
+    3. ReturnIfAbrupt(fromValue).
+    4. Let excludedNames be a new empty List.
+    5. Return CopyDataProperties(object, fromValue, excludedNames).
+
+---*/
+
+//- args
+{a: 1, b: 2, ...null}
```

Comment:
Many of the tests in this PR seem to be wrong, based on what is in the spec. I pointed this out on https://github.com/tc39/ecma262/pull/1048#discussion_r200538519

I understand that there's been some expectation that it should behave like this, but it's not what's specified, due to the way CopyDataProperties is specified normatively.

I think we should either update the spec, or update these tests.